numbers.js

```diff
@@ -0,0 +1,59 @@
+console.log('num1+num2 represents the equation to find the sum of these numbers');
+console.log('num1-num2 gives you the difference between these numbers');
+console.log('num1*num2 gives you the product of two numbers');
+console.log('num1/num2 divides the 1st number by the 2nd');
+console.log('num1%num2 reveals if there is a remainder after the numbers are divided');
+console.log('num1**num2 put the 2nd number in the sequence as an exponent');
+console.log('Math.floor(num1/num2) rounds down the quotient');
+
+function evenorodd (blah){
+if (blah%2===0){
+return 'even' ;
+} else {
+  return 'odd';
+}
+}
+//Tried this at first, didn't work
+  //function increments (blah){
+  //let word = blah;
+  //word+=1;
+  //console.log(word);
+  //word = blah;
+  //word+=2;
+  //console.log(word);
+  //word = blah;
+  ///word+=10;
+  //console.log(word);
+  //word = blah;
+  //word-=1;
+  //console.log(word);
+  //word = blah;
+  //word-=2;
+  //console.log(word);
+  //word = blah;
+  //word-=10;
+ // return word;
+  //}
+  //function increments2 (k, blah) {
+   // blah+=k;
+   // console.log(blah);
+    //blah-=k;
+    //return blah;
+
+  //}
+
+  //console.log(evenorodd(50));
+  //console.log(increments(50));
+  //console.log(increments2(10,50));
+  function increments (a,b) {
```

Comment:
Keep it simple. Without writing or using any functions, what can you put in the `______` to increment the value of `someNumber` by `1`?

```javascript
let someNumber = 108;
console.log('The value of someNumber is:', someNumber);
______;
console.log('After incrementing by 1, the value of someNumber is:', someNumber);
```

You don't need to do the incrementing inside a function. Don't use a function if it trips you up.